spec/models/picture_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Picture, type: :model do
+  it { should have_attached_file(:image) }
```

Comment:
this is the old way of doing single line rspec methods, see if you can find the new way of doing it